src/components/Tabset/readme.md

```diff
@@ -791,3 +791,155 @@ class TabsetExample extends React.Component {
 
 <TabsetExample />;
 ```
+
+##### Tabset linear
+
+```js
+import React from 'react';
+import { Tabset, Tab } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const StyledTabLabel = styled.div`
+    font-family: Lato;
+    font-size: 14px;
+    letter-spacing: 0.46px;
+`;
+
+const StyledContainer = styled.div`
+    max-width: 600px;
+`;
+
+const StyledContent = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    background: ${props => props.background.main};
+    width: 100%;
+    border-radius: 0.875rem 0.875rem;
+`;
+
+const StyledTabContentTitle = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.main};
+    font-family: Lato;
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 0.83;
+`;
+
+const StyledTabContentText = styled.div`
+    font-family: Lato;
+    font-size: 14px;
+    line-height: 1.43;
+`;
+
+const StyledIcon = styled.div`
+    width: 60px;
+    height: 50px;
+    border-radius: 100px;
+    background-color: rgba(227, 231, 233, 0.25);
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+`;
+
+const facebookIconStyles = { color: '#3c5997', height: 30 };
+const twitterIconStyles = { color: '#00b0f3', width: 30 };
+const googleIconStyles = { height: 30 };
+
+class TabsetExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { selected: 'companies' };
+        this.handleOnSelect = this.handleOnSelect.bind(this);
+    }
```

Comment:
we should add new examples using functional components with hooks instead of class components